modules/nw-osp-pod-connections-ipv6.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-connections-ipv6_{context}"]
+= Enabling IPv6 connectivity to pods on {rh-openstack}
+
+To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security obviates the need to create allowed address pairs.
+
+[IMPORTANT]
+====
+Only the following IPv6 additional network configurations are supported:
+
+* SLAAC and host-device
+* SLAAC and MACVLAN
+* DHCP stateless and host-device
+* DHCP stateless and MACVLAN
+====
+
+.Procedure
+
+* On a command line, enter:
++
+[source,terminal]
+----
+$ openstack port set --no-security-group --disable-port-security <compute_ipv6_port>
+----
```

Comment:
I would add the following note:

Since traffic is allowed, private addresses are recommended to be used.

@mandre what do you think about adding this NOTE?

Reply:
maybe we can mention something about private ipv6 addresses as well?

Reply:
"would," "should," or more like "Traffic restrictions must not be imposed on that port."

Reply:
this NOTE is to alert about what that command is actually doing

Reply:
Yes, we should highlight that disabling the port security has security implication, so that customers know what the command is doing.

Reply:
```suggestion
----
+
IMPORTANT: This command removes security groups from the port and disables port security. Traffic restrictions are removed entirely from the port.  
```

So, something like this?

Reply:
Yes.